projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -1066,46 +1166,79 @@ function register_settings() {
 		register_setting( 'media', 'carousel_display_comments', array( $this, 'carousel_display_comments_sanitize' ) );
 
-		// No geo setting yet, need to "fuzzify" data first, for privacy
-		// add_settings_field('carousel_display_geo', __( 'Geolocation', 'jetpack' ), array( $this, 'carousel_display_geo_callback' ), 'media', 'carousel_section' );
-		// register_setting( 'media', 'carousel_display_geo', array( $this, 'carousel_display_geo_sanitize' ) );
 	}
 
-	// Fulfill the settings section callback requirement by returning nothing
-	function carousel_section_callback() {
-		return;
+	/**
+	 * Fulfill the settings section callback requirement by returning nothing.
+	 */
+	public function carousel_section_callback() {
 	}
 
-	function test_1or0_option( $value, $default_to_1 = true ) {
-		if ( true == $default_to_1 ) {
-			// Binary false (===) of $value means it has not yet been set, in which case we do want to default sites to 1
+	/**
+	 * Tests if a value is set
+	 *
+	 * @param mixed $value The value passed into this function with which to test.
+	 * @param bool  $default_to_1 Default is true.
+	 *
+	 * @return bool
+	 */
+	public function test_1or0_option( $value, $default_to_1 = true ) {
+		if ( true === $default_to_1 ) {
+			// Binary false (===) of $value means it has not yet been set, in which case we do want to default sites to 1.
 			if ( false === $value ) {
 				$value = 1;
 			}
 		}
-		return ( 1 == $value ) ? 1 : 0;
+		return ( 1 === $value ) ? 1 : 0;
 	}
 
-	function sanitize_1or0_option( $value ) {
-		return ( 1 == $value ) ? 1 : 0;
+	/**
+	 * Ensures the value returned is in the correct format.
+	 *
+	 * @see test_1or0_option()
+	 * @param mixed $value The value returned from the test_1or0_option function.
+	 *
+	 * @return int
+	 */
+	public function sanitize_1or0_option( $value ) {
+		return ( 1 === $value ) ? 1 : 0;
 	}
 
-	function settings_checkbox( $name, $label_text, $extra_text = '', $default_to_checked = true ) {
+	/**
+	 * Adds settings checkbox
+	 *
+	 * @param string $name - For name attribute.
+	 * @param string $label_text - For label attribute.
+	 * @param string $extra_text - Additional checkbox description text. Defaults to empty.
+	 * @param bool   $default_to_checked - If the checkbox is checked. Default is true.
+	 *
+	 * @return string
+	 */
+	public function settings_checkbox( $name, $label_text, $extra_text = '', $default_to_checked = true ) {
 		if ( empty( $name ) ) {
 			return;
 		}
 		$option = $this->test_1or0_option( get_option( $name ), $default_to_checked );
 		echo '<fieldset>';
 		echo '<input type="checkbox" name="' . esc_attr( $name ) . '" id="' . esc_attr( $name ) . '" value="1" ';
 		checked( '1', $option );
-		echo '/> <label for="' . esc_attr( $name ) . '">' . $label_text . '</label>';
+		echo '/> <label for="' . esc_attr( $name ) . '">' . esc_html( $label_text ) . '</label>';
```

Comment:
Looks like `$label_text` is intended to contain HTML, see for example line 1264.

We could either leave it as it was, or do some sort of `wp_kses()` (maybe `wp_kses_allowed_html()`?) to limit the allowed HTML.

Reply:
Oh yeah. Perhaps `wp_kses_post` would be ok here too?